torch/_dynamo/variables/builtin.py

```diff
@@ -507,6 +559,35 @@ def call_enumerate(self, tx, *args):
             ]
             return variables.TupleVariable(items, **options)
 
+    def _call_operator_builtin(self, tx, op_name: str, *args, **kwargs):
+        options = VariableTracker.propagate(self, args)
+        fn_ = getattr(operator, op_name)
+
+        # This should really be refactored.
+        # builtins like + and - can return a ton of different types,
+        # depending on the types of the inputs.
+        # Right now, TensorVariable.create() handles a subset of these cases,
+        # but not all of them.
+        # For example, different cases that we care about include
+        # when the output type is a:
+        # - ConstantVariable (handled by ConstantVariable.call_method)
+        # - DynamicShapesVariable (handled by TensorVariable.create)
+        # - TensorVariable (handled by TensorVariable.create)
+        if all(isinstance(x, variables.ConstantVariable) for x in args):
+            return args[0].call_method(tx, f'__{op_name}__', args[1:], kwargs)
+
+        out = TensorVariable.create(
```

Comment:
This isn't right, I talked to @ezyang about it in detail.

Reply:
The issue is you need to do something different depending on the type.  For example `ListVariable.call_method`, `NNModuleVariable.call_method`, etc.

Reply:
+1 we should handle this differently